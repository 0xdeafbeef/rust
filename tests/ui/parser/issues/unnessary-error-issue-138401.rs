pub fn foo(x: i64) -> i64 {
    x.abs) 
    //~^ ERROR mismatched closing delimiter
}
//~^ ERROR unexpected closing delimiter: `}`

fn main() {}
