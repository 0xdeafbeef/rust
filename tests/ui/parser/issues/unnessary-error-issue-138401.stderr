error: mismatched closing delimiter: `)`
  --> $DIR/unnessary-error-issue-138401.rs:1:27
   |
LL | pub fn foo(x: i64) -> i64 {
   |                           ^ unclosed delimiter
LL |     x.abs) 
   |          ^ mismatched closing delimiter

error: unexpected closing delimiter: `}`
  --> $DIR/unnessary-error-issue-138401.rs:4:1
   |
LL |     x.abs) 
   |          - missing open `(` for this delimiter
LL |
LL | }
   | ^ unexpected closing delimiter

error: aborting due to 2 previous errors

